build/Common.tests.props

```diff
@@ -37,19 +37,25 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
+
+        <!-- Fare and System.Text.RegularExpressions are referenced directly to avoid indirect vulnerable packages from AutoFixture -->
+        <PackageReference Include="Fare" Version="$(FareVer)" />
+        <PackageReference Include="System.Text.RegularExpressions" Version="$(SystemTextRegularExpressionsVer)" />
```

Comment:
In the `SDK`, we migrated to the preview version to avoid the same problem you are facing. https://github.com/open-feature/dotnet-sdk/blob/1acc00fa7a6a38152d97fd7efc9f7e8befb1c3ed/Directory.Packages.props#L24

Both are valid, and I am happy to pin these references.

Reply:
It is up to you, I can change to the preview, There were no references to unstable packages, so that decided to manually bump.

Reply:
Let's not change here. When we bump to v5, we must remove the version pinning. 👍